src/ncp/spinel.h

```diff
@@ -383,7 +383,7 @@ enum
 
     SPINEL_CAP_EXPERIMENTAL__BEGIN      = 2000000,
     SPINEL_CAP_EXPERIMENTAL__END        = 2097152,
-};
+} spinel_capability_t;
```

Comment:
Changing this to an enum makes it more clumsy for vendors to define their own capabilities without editing this file. Are we sure we want to do this?